src/main/java/org/bahmni/reports/BahmniReportsProperties.java

```diff
@@ -143,4 +143,9 @@ public String getMartPassword() {
     public String getFileUploadMaxSize() {
         return props.getProperty("reports.config.fileUploadMaxSize");
     }
+
+    public String getReportsTimeZone() {
+        return props.getProperty("reports.timezone");
```

Comment:
I feel that the Report Properties should handle and return either the "Set" value or the ServerTimezone. This way, any module that calls the Report Properties to getTimezone will always get a value. 

If the Header does this logic, then it becomes Header specific. Whereas, this should be the same logic for any module that wants to know the report module. 

@shilpa-iplit - can u move this logic from header into ReportProperties?

Reply:
Done. Please review. Thanks.